Source/Flow/Public/Nodes/FlowNode.h

```diff
@@ -332,6 +332,10 @@ class FLOW_API UFlowNode : public UObject, public IVisualLoggerDebugSnapshotInte
 	UFUNCTION(BlueprintImplementableEvent, Category = "FlowNode", meta = (DisplayName = "Cleanup"))
 	void K2_Cleanup();
 
+	// Method called from UFlowAsset::DeinitializeInstance() 
+	//  (ie, when deinitializing the flow asset itself)
+	virtual void DeinitializeInstance() { }
```

Comment:
Base-class DeinitializeInstance function for flownode does nothing (but subclasses might want to do stuff)

Reply:
Nice one :)
Added here calling a blueprint event `K2_DeinitializeInstance` by default. I suspect that someone would request it at some point.